src/main/scala/org/broadinstitute/dsde/firecloud/webservice/CromIamApiService.scala

```diff
@@ -14,15 +13,20 @@ trait CromIamApiService extends FireCloudRequestBuilding
   lazy val workflowRoot: String = FireCloudConfig.CromIAM.authUrl + "/workflows/v1"
   lazy val womtoolRoute: String = FireCloudConfig.CromIAM.authUrl + "/womtool/v1"
   lazy val engineRoot: String = FireCloudConfig.CromIAM.baseUrl + "/engine/v1"
-//  lazy val rawlsWorkflowRoot: String = FireCloudConfig.Rawls.authUrl + "/workflows"
+  lazy val rawlsWorkflowRoot: String = FireCloudConfig.Rawls.authUrl + "/workflows"
 
   // This is the subset of CromIAM endpoints required for Job Manager. Orchestration is acting as a proxy between
   // CromIAM and Job Manager as of February 2019.
   // Adam Nichols, 2019-02-04
 
-  val cromIamApiServiceRoutes: Route =
-    pathPrefix( "workflows" / Segment ) { _ =>
-      streamingPassthrough(Uri.Path(s"/api/workflows/v1") -> Uri(workflowRoot))
+  val cromIamApiServiceRoutes: Route = {
+    val localBase = s"/api/workflows/v1"
+    pathPrefix("workflows" / Segment / Segment / "backend" / "metadata" / Segments) {(version, workflowId, operationSegments) =>
+      val suffix = operationSegments.mkString("/")
+      passthroughImpl(Uri.Path(localBase), Uri(rawlsWorkflowRoot), Option(s"/${workflowId}/genomics/${suffix}"))
+    } ~
+    pathPrefix( "workflows" / Segments ) { segmentsArr =>
+      streamingPassthrough(Uri.Path(localBase) -> Uri(workflowRoot))
```

Comment:
Yeah, that makes sense. The main reason I want to avoid a separate file is because eventually the new file will need to be imported and referenced in `FireCloudApiService`. Due to the overlapping root path structure, I'm sure that the order in which the files are defined there will matter

Not that it won't be an issue here in `CromIamService`, but at lease the context behind the issue is defined in the same spot rather than elsewhere.